Specification/BrAPI-Schema/BrAPI-Phenotyping/Scale.json

```diff
@@ -101,9 +95,9 @@
                 "primaryModel": true
             }
         },
-        "ValidValues": {
+        "ValidValue": {
```

Comment:
I actually think that ValidValues is correct over ValidValue in this case. The property would be validValues and it references 'one-to-one' an object of class 'ValidValues'